src/main/java/digital/slovensko/autogram/core/visualization/DocumentVisualizationBuilder.java

```diff
@@ -66,7 +66,7 @@ private Visualization createVisualization(SigningJob job, MimeType transformatio
         if (document.getMimeType().equals(MimeTypeEnum.HTML)) {
             return new HTMLVisualization(transform(), job);
         } else if (document.getMimeType().equals(MimeTypeEnum.TEXT)) {
-            return new PlainTextVisualization(transform(), job);
+            return new PlainTextVisualization(new String(document.openStream().readAllBytes()), job);
```

Comment:
Toto ale vyzera mimoriadne podozrivo. Plaintext neprachadza cez transform?

Reply:
nie. Transform metóda slúži na transformáciu cez XSLT. Vyššie v kóde sa rieši, že ak je to XDC alebo XML+XSLT, tak sa transformuje. Tu sa už žiadny transform nedá robiť s obyčajným textom. Potom asi vyhodíme aj to HTML nad tým, pretože to je vetva pre zobrazenie HTML súboru, nie HTML transformácie. A HTML sme nespomínali, že by sme chceli zobrazovať, lebo to nemá usecase.

Reply:
Ok dajme tam ale nejake TODO nech sa nezabudne.